tasks/gulp/backstop-config.js

```diff
@@ -86,4 +86,9 @@ module.exports = ({ host, port, components }) => ({
   asyncCompareLimit: 50,
   debug: false,
   debugWindow: false,
+  misMatchThreshold: 0,
+  resembleOutputOptions: {
+    ignoreAntialiasing: true,
+    usePreciseMatching: true
+  }
```

Comment:
otherwise it seems lots of changes aren't big enough compared to the size of the screenshot to actually show up as a change. If you look at the updated backstop screenshots you'll find underlines, spacing changes, and even one case where our reference image was a serif font and the test image was a sans-serif font but it showed as no changes...